napalm_iosxr/__init__.py

```diff
@@ -13,4 +14,14 @@
 # the License.
 
 """napalm_iosxr package."""
-from iosxr import IOSXRDriver
+
+# Import stdlib
+import pkg_resources
+
+# Import local modules
+from iosxr import IOSXRDriver  # noqa
```

Comment:
Thanks for your suggestions! I've updated as per your comments and I made some more cleanup especially under ```get_interfaces_ip```